domains/authorization/Authorization.API/API/Migrations/ApplicationDbContextModelSnapshot.cs

```diff
@@ -115,9 +106,6 @@ protected override void BuildModel(ModelBuilder modelBuilder)
 
                     b.HasKey("Id");
 
-                    b.HasIndex("IdpOrganizationId")
-                        .IsUnique();
```

Comment:
No index?

Also why remove uniqueness constraint?

Reply:
The property is removed